### PR TITLE
Run marathon as a docker container.

### PR DESCRIPTION
**Pull request by enxebre**

### CODE REVIEW

File: roles/marathon/defaults/main.yml [hunk 1]
```diff
@@ -1,8 +1,30 @@
 ---
 # defaults file for marathon
-marathon_port: 8080
-consul_dir: /etc/consul.d
+marathon_enabled: true
+marathon_version: 'v0.8.2'
```
Comment (tayzlor) [inline roles/marathon/defaults/main.yml]: I think we should use https://github.com/mesosphere/marathon/releases/tag/v0.9.0-RC2

File: roles/marathon/tasks/main.yml [hunk 2]
```diff
@@ -1,104 +1,81 @@
...
-    src: marathon.conf.j2
-    dest: /etc/init/marathon.conf
-    mode: 0755
-  notify:
-    - restart marathon
-  sudo: yes
-  tags:
-    - marathon
+# Install (docker-py) python package as is a docker module dependency.
+- pip: name=docker-py version=1.1.0
```
Comment (tayzlor) [inline roles/marathon/tasks/main.yml]: dont think this is needed now its installed in packer?

**Comment by tayzlor** [comment]
This looks pretty good. after the comments and a sqaush it should be good to merge.

**Comment by tayzlor** [comment]
should we be removing the packer install scripts for the package in this PR too?

**Comment by tayzlor** [comment]
Lets also tag on master before this gets in

File: roles/marathon/defaults/main.yml (hunk 1 above)
Comment (enxebre) [inline roles/marathon/defaults/main.yml]: Would need to rebuild packer images if we dont want it to go the internet though

**Comment by enxebre** [comment]
I will tag, then I would keep the packages in packer and merge this PR. I'll create a follow up for removing the packages and lastest marathon image version.
https://github.com/Capgemini/Apollo/issues/339

**Comment by enxebre** [comment]
Also I noticed some perfomance issue when running marathon inside the container in vagrant that need to be investigated.

File: roles/marathon/tasks/main.yml (hunk 2 above)
Comment (enxebre) [inline roles/marathon/tasks/main.yml]: Probablt dont neede but I like it as It helps to keep the role more self-contained

**Pull request merged**